BDArmory/Radar/RadarUtils.cs

```diff
@@ -2651,18 +2583,18 @@ public static bool checkWater(Vector3 start, Vector3 end, CelestialBody body, fl
         /// </summary>
         public static Vector2 WorldToRadar(Vector3 worldPosition, Transform referenceTransform, Rect radarRect, float maxDistance)
         {
-            float scale = maxDistance / (radarRect.height / 2);
+            float scale = (radarRect.height * 0.5f) / maxDistance;
             Vector3 localPosition = referenceTransform.InverseTransformPoint(worldPosition);
             localPosition.y = 0;
             if (BDArmorySettings.LOGARITHMIC_RADAR_DISPLAY)
             {
-                scale = Mathf.Log(localPosition.magnitude / scale + 1) / Mathf.Log(radarRect.height + 1);
+                scale = Mathf.Log(localPosition.magnitude * scale + 1) / Mathf.Log(radarRect.height + 1);
                 localPosition = localPosition.normalized * scale;
-                return new Vector2(radarRect.width * (1 + localPosition.x) / 2, radarRect.height * (1 - localPosition.z) / 2);
+                return new Vector2((radarRect.width * 0.5f) * (1 + localPosition.x), (radarRect.height * 0.5f) * (1 - localPosition.z));
             }
             else
             {
-                return new Vector2((radarRect.width / 2) + (localPosition.x / scale), ((radarRect.height / 2) - (localPosition.z / scale)));
+                return new Vector2((radarRect.width * 0.5f) + (localPosition.x * scale), ((radarRect.height * 0.5f) - (localPosition.z * scale)));
```

Comment:
This should still be `/ scale` as it's using the new value of scale calculated on line 2591, which is no longer inverted.